data-raw/process_data_sets.R

```diff
@@ -272,3 +272,19 @@ pennies_resamples <-
   select(replicate, everything()) %>%
   unnest(cols = c(data))
 usethis::use_data(pennies_resamples, overwrite = TRUE)
+
+#-------------------------------------------------------------------------------
+# Pennies: Documented in R/datasets.R
+#-------------------------------------------------------------------------------
+## Population of 143 Ebay auctions selling Mario Kart for Nintendo Wii
+## Original Google Sheet here:
+## https://docs.google.com/spreadsheets/d/1jhiTFaaJ4ZCUA9yMNFGQ2xnKItXewaNmzu6y0Syw_dk/edit
+library(tidyverse)
```

Comment:
Please remove the call to `library()`, because `tidyverse` is also loaded at the top of the file.